### DISCUSSION TITLE
Will Phoenix 6 support Talon SRX??

### DISCUSSION THREAD

**Discussion by kglasnapp** [wpilibsuite/2024Beta#58] [Q&A] [answered]
Will Phoenix 6 support Talon SRX??

**Comment by ThadHouse** [comment, marked as answer]
https://store.ctr-electronics.com/phoenix-6-announcement-details/

In their announcement,

"Legacy classes in Phoenix 5, such as TalonSRX and VictorSPX, will not be moving to Phoenix 6 and will remain available as-is in Phoenix 5."
